### DISCUSSION TITLE
음악 검색 시스템 만들기 | kciter.so

### DISCUSSION THREAD

**Discussion by giscus** [kciter/kciter.github.io#37] [Announcements]
# 음악 검색 시스템 만들기 | kciter.so

미래 학자이자 SF 소설의 거장 아서 C. 클라크는 라는 유명한 말을 남겼다. 이 말이 사실임을 증명하듯 현대 사회의 기술은 다양한 놀라움을 선사한다. 그 중에서도 Shazam이나 Soundhound와 같은 음악 검색 앱을 사용할 때 필자는 항상…

https://kciter.so/posts/music-recognition-system/

**Comment by radiantchoi** [comment]
ShazamKit을 써서 음악 검색 앱을 만들어 본 입장에서, 내부 동작에 대해서는 WWDC 등에서 보고 어렴풋이 알고 있는 정도였는데, 이렇게 자세히 설명해주신 글을 읽다 보니 매우 흥미로웠습니다. 좋은 글 감사드립니다.

**Reply by kciter** [reply]
좋은 말씀 감사합니다! 앞으로도 열심히 써보겠습니다. 😄

**Comment by greenheadHQ** [comment]
FFT의 존재만 알고 어떻게 활용되는 지에 대해서는 잘 모르고 있었는데, 이렇게도 쓰인다는 점은 정말 흥미롭네요. 좋은 글 감사합니다! :)

**Comment by roeniss** [comment]
이 시리즈 너무 좋네요
